packages/stylelint-plugin/src/index.ts

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2020 Palantir Technologies, Inc. All rights reserved.
```

Comment:
nit: formatting of block comment is slightly off (is the autofixer broken?)

Reply:
I fixed this manually, but actually it seems like this is also a valid format and the autofixer always does it this way on my computer. Could you test what happens on your machine?

Reply:
It looks like the plugin only inserts starts at the beginning and end https://github.com/Stuk/eslint-plugin-header/blob/66f5269c8a7e3282afd02b914b502f7bb95fe702/lib/rules/header.js#L46

Do you have a template for ts/js files configured in VS code by any chance?